department-of-veterans-affairs/va.gov-cms
# VAGOV-3812

**About:**
This pr adds a new custom module that leverages the feature_toggle module.
The idea is that users can go `/admin/config/system/feature_toggle`, add a feature flag (used for preventing graphql items in front end code throwing errors when out of sync with Drupal endpoint), and enable or disable per environment. This will prevent multiple code update requirement that we currently have to disable / enable flags in various environments.

Logic has not been built into vets-website (yet). This is step one.

**QA:**
- Go to `/admin/config/system/feature_toggle` and add some flags (toggle some on, leave some untoggled)
- Go to `/flags_list` and verify that the flags you added show and reflect the true/false status you set on the form.

**Screenshots:**
![Screenshot_2019-06-19_11-59-01](https://user-images.githubusercontent.com/2404547/59781401-b36dc080-9289-11e9-9346-b5cd4fb439ab.png)
![Screenshot_2019-06-19_11-58-50](https://user-images.githubusercontent.com/2404547/59781404-b36dc080-9289-11e9-900c-85212730ad13.png)
